app/models/account.rb

```diff
@@ -36,6 +36,7 @@
 #  followers_count         :integer          default(0), not null
 #  following_count         :integer          default(0), not null
 #  last_webfingered_at     :datetime
+#  activitystreams2_url    :string
```

Comment:
Yeah I thought about that option, but *technically* AP is only the protocol for server/client communication, built on top of AS2 representations. The URL is the user's AS2 representation (which happens to implement AP), so that seemed like the more accurate name. Not a big deal either way though.